documentation/style-guide.rst

```diff
@@ -196,6 +199,34 @@ language):
     achieve the same effect.  This assures that files are flushed and file
     descriptor resources are released in a timely manner.
 
+
+Bylines
+=======
+
+For new documentation, do not use bylines. They tend to discourage other users
```

Comment:
It's not clear to me what a "byline" is. Do you mean author attribution?

Reply:
Good point, I've updated it.

Reply:
It's much clearer like this, thank you!